velox/expression/fuzzer/ExpressionFuzzer.h

```diff
@@ -26,6 +26,14 @@
 #include "velox/vector/tests/utils/VectorMaker.h"
 
 namespace facebook::velox::fuzzer {
+namespace detail {
```

Comment:
APIs in 'detail' namespaces are not supposed to be used outside of the header/cpp files.

I wonder if this API belongs to velox/functions/FunctionRegistry.h and whether it is clearer to return std::optional<bool> to indicate not-found condition clearly.